examples/user_defined_iface.yml

```diff
@@ -0,0 +1,43 @@
+---
+user-defined:
+    interface-types:
+        - name: vxcan
+          handler-script: |
```

Comment:
Just using `handler` might be good enough and allows to simplify the state (no dash) to make it easier to type manually.